lib/manageiq_performance/middlewares/active_record_queries.rb

```diff
@@ -130,12 +132,19 @@ def should_measure?
           Thread.current[:miq_perf_sql_query_data]
         end
 
+        def include_queries?
+          return @include_queries if defined?(@include_queries)
+          @include_queries = ManageIQPerformance.config.include_sql_queries?
```

Comment:
if `@include_queries == false`, then `||=` will access `config` every time.

So we can either remove the `@` variable, or we want to cache it whether it is `true` or `false`